src/core/sys/linux/epoll.d

```diff
@@ -142,7 +147,84 @@ union epoll_data_t
     ulong u64;
 }
 
+/**
+ * Creates an epoll instance.
+ *
+ * Params:
+ *   size = a hint specifying the number of file descriptors to be associated
+ *          with the new instance.  T
+ * Returns: an fd for the new instance. The fd returned by epoll_create() should
+ *          be closed with close().
+ * See_also: epoll_create1 (int flags)
+ */
 int epoll_create (int size);
+
+/* Same as epoll_create but with an FLAGS parameter.  The unused SIZE
+   parameter has been dropped.  */
+
+/**
+ * Creates an epoll instance.
+ *
+ * Params:
+ *   flags = a specified flag. If flags is 0, then, other than the fact that the
+ *           obsolete size argument is dropped, epoll_create1() is the same as
+ *           epoll_create().
+ * Returns: an fd for the new instance. The fd returned by epoll_create() should
+ *          be closed with close().
+ * See_also: epoll_create (int size)
+ */
 int epoll_create1 (int flags);
+
+/**
+ * Manipulate an epoll instance
+ *
+ * Params:
+ *   epfd = an epoll file descriptor instance
+ *   op = one of the EPOLL_CTL_* constants
+ *   fd = target file descriptor of the operation
+ *   event = describes which events the caller is interested in and any
+ *           associated user dat
+ * Returns: 0 in case of success, -1 in case of error ( the "errno" variable
+ *          will contain the specific error code )
+ */
 int epoll_ctl (int epfd, int op, int fd, epoll_event *event);
+
+
+/**
+ * Wait for events on an epoll instance.
+ *
+ * This function is a cancellation point and therefore not marked with __THROW.
+ *
+ * Params:
+ *   epfd = an epoll file descriptor instance
+ *   events = a buffer that will contain triggered events
+ *   maxevents = the maximum number of events to be returned ( usually size of
+ *               "events" )
+ *   timeout = specifies the maximum wait time in milliseconds (-1 == infinite)
+ *
+ * Returns: the number of triggered events returned in "events" buffer. Or -1 in
+ *          case of error with the "errno" variable set to the specific error
+ *          code.
+ */
 int epoll_wait (int epfd, epoll_event *events, int maxevents, int timeout);
+
+/**
+ * Wait for events on an epoll instance
+ *
+ * This function is a cancellation point and therefore not marked with __THROW.
+ *
+ * Params:
+ *   epfd = an epoll file descriptor instance
+ *   events = a buffer that will contain triggered events
+ *   maxevents = the maximum number of events to be returned ( usually size of
+ *               "events" )
+ *   timeout = specifies the maximum wait time in milliseconds (-1 == infinite)
+ *   ss = a signal set. May be specified as NULL, in which case epoll_pwait() is
```

Comment:
Likewise, D doesn't have `NULL`.  Though you could use `` `null` `` instead.

Reply:
Done